packages/baby-jubjub/rollup.config.ts

```diff
@@ -15,24 +17,45 @@ const banner = `/**
 
 const name = pkg.name.split("/")[1].replace(/[-/]./g, (x: string) => x.toUpperCase()[1])
 
-export default {
-    input: "src/index.ts",
-    output: [
-        { file: pkg.exports["."].require, format: "cjs", banner },
-        { file: pkg.exports["."].default, format: "es", banner },
-        {
-            file: pkg.iife,
-            name,
-            format: "iife",
-            banner
-        },
-        {
-            file: pkg.unpkg,
-            name,
-            format: "iife",
-            plugins: [terser({ output: { preamble: banner } })]
-        }
-    ],
-    external: Object.keys(pkg.dependencies),
-    plugins: [typescript({ tsconfig: "./build.tsconfig.json" }), cleanup({ comments: "jsdoc" })]
-}
+export default [
+    {
+        input: "src/index.ts",
+        output: [
+            { file: pkg.exports["."].require, format: "cjs", banner },
+            { file: pkg.exports["."].default, format: "es", banner }
+        ],
+        external: [
+            ...Object.keys(pkg.dependencies),
+            "@zk-kit/utils/conversions",
+            "@zk-kit/utils/f1-field",
+            "@zk-kit/utils/scalar"
+        ],
+        plugins: [typescript({ tsconfig: "./build.tsconfig.json" }), cleanup({ comments: "jsdoc" })]
+    },
+    {
+        input: "src/index.ts",
+        output: [
+            {
+                file: pkg.iife,
+                name,
+                format: "iife",
+                banner
+            },
+            {
+                file: pkg.unpkg,
+                name,
+                format: "iife",
+                plugins: [terser({ output: { preamble: banner } })]
+            }
+        ],
+        external: [],
+        plugins: [
+            typescript({
+                tsconfig: "./build.tsconfig.json"
+            }),
+            nodeResolve({ preferBuiltins: false, browser: true }),
+            nodePolyfills(),
```

Comment:
Why do we need the polyfills here? Is this a problem related to the `buffer`?

Reply:
Yes, it looks like Rollup still wants the polyfill for `buffer`, even if we're using a package (from which polyfill is based I think). I've tried with `commonjs` and `node-resolve` but they're not enough.